docs/layers.md

```diff
@@ -839,9 +844,11 @@ To improve performance, some road segments are merged at low and mid-zooms. To f
 * `leisure`: See kind list below.
 * `man_made`: See kind list below.
 * `network`: eg: `US:I` for the United States Interstate network, useful for shields and road selections.
+* `oneway_bicycle`: `oneway:bicycle` tag from feature
 * `oneway`: `yes` or `no`
 * `piste_type`: See kind list below.
 * `railway`: the original OSM railway tag value
+* `segregated`: `segregated` tag from feature.
```

Comment:
Is `segregated` for bicycle cycleways? If so, please update documentation to clarify that.